functions.php

```diff
@@ -1122,10 +1123,13 @@ function eypd_d3_array( $data ) {
 			unset( $unit );
 
 		}
+		if ( 0 === strcmp( 'Professional Development Certificate Credit Hours', $key ) ) {
```

Comment:
`$key` is undefined at this point, as is `$total`, which is never referenced again - can you indicate what purpose this serves?

Reply:
abandoned train of thought that I didn't remove. 1104 can go and 1126-1128